src/org/omegat/Bundle.properties

```diff
@@ -286,6 +286,7 @@ TF_FILE_IMPORT_TITLE=Add Files to Source Folder...
 TF_MENU_WIKI_IMPORT=Add MediaWiki Pa&ge...
 TF_WIKI_IMPORT_TITLE=Add MediaWiki Page
 TF_WIKI_IMPORT_PROMPT=URL
+TF_WIKI_IMPORT_URL_ERROR=Provided URL is invalid.
```

Comment:
```suggestion
TF_WIKI_IMPORT_URL_ERROR=Provided URL is invalid.
TF_WIKI_IMPORT_URL_ERROR_TITLE=Url Error
```

Reply:
Do we use URL or URI ?

Reply:
Input dialog use "URL" for prompt.

```
TF_MENU_WIKI_IMPORT=Add MediaWiki Pa&ge...
TF_WIKI_IMPORT_TITLE=Add MediaWiki Page
TF_WIKI_IMPORT_PROMPT=URL
```

Reply:
Yes. I thought URI was a better choice for our UI in general but that does not seem to be the case. So let's leave URL.